SC_redesign/Crop_and_Soil/manager/output_gatherer.py

```diff
@@ -240,6 +240,70 @@ def send_daily_variables(self, filter_specs: Optional[List] = None) -> None:
                 om.add_variable("residue_nitrogen", crop.data.residue_nitrogen, info_map)
                 om.add_variable("residue_phosphorus", crop.data.residue_phosphorus, info_map)
 
-    def send_annual_variables(self):
+    def send_annual_variables(self, filter_specs: Optional[List] = None) -> None:
```

Comment:
```suggestion
    def send_annual_variables(self) -> None:
```
I think we are good to not include any kind of filter because `OutputManager` will handle all filtering for us.